src/patterns/validation/index.md.njk

```diff
@@ -67,11 +67,13 @@ There are 2 ways to carry out validation:
 + server side validation
 + client side validation (in the browser, using either JavaScript or HTML)
 
-You’ll always need to carry out server side validation, because there’s no guarantee that client side validation will work in all circumstances. For example users can bypass client side validation, or [JavaScript can fail to load](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).
+You’ll always need to carry out server side validation.
+
+If you do use client side validation, you'll still need to carry out server side validation as well. This is because there’s no guarantee that client side validation will work in all circumstances. For example users can bypass client side validation, or [JavaScript can fail to load](https://www.gov.uk/service-manual/technology/using-progressive-enhancement).
```

Comment:
If the suggestion above could work, then we can delete this line.

Reply:
Agreed